controller/openebs/common.go

```diff
@@ -85,6 +85,12 @@ type BasicComponentDetails struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 }
 
+// supportRCImagesForOpenEBS1100EE supports installation of OpenEBS 1.10.0-ee RC images.
+// TODO: Remove this once 1.10.0-ee images are available.
+func (p *Planner) supportRCImagesForOpenEBS1100EE() {
+	p.ObservedOpenEBS.Spec.Version = "1.10.0-ee-RC3"
```

Comment:
Can this be done via a label or something. Avoid code addition and removal.

Reply:
Done